model_compiler/setup.py

```diff
@@ -74,7 +74,9 @@ def run(self):
             'mypy',
             'pylint==2.6.2',
             'pytest-cov',
-            'pytest-xdist'
+            'pytest-xdist',
+            'types-requests',
```

Comment:
Do not add `pytest-xdist` and `types-requests` to test deps.
Rebase master for a workaround for now.